test/root.js

```diff
@@ -17,4 +17,6 @@ before(async () => {
 after(async () => {
   await expressServer.close();
   console.log("stopped express server");
+
+  // TODO stop cytosnap instance(s)
```

Comment:
@mikekucera, is there a simple way to do this now?  If not, let's leave this TODO for later.

Reply:
OK, it sounds like we should just rely on `--exit` for now.

If there's crash, then we should make an issue for that to investigate/prioritise it.  It sounds like something like that should be resolved before those features make it to master.  Maybe there's a memory issue or something simple.  We'll have to investigate and see

Reply:
See #143